### PR TITLE
Pyclustering

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyclustering/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  #   - {{ compiler('c') }}
+  #   - core
+  missing_dso_whitelist:
+    - /lib64/libm.so.6  # [linux]
+    - /lib64/libstdc++.so.6  # [linux]
+    - /lib64/ld-linux-x86-64.so.2  # [linux]
+    - /lib64/libgcc_s.so.1  # [linux]
+    - /lib64/libc.so.6  # [linux]
+    - /lib64/libSystem.B.dylib  # [osx]
+    - /lib64/ld-linux.so.2  # [linux]
```
Comment (msarahan) [inline recipes/pyclustering/meta.yaml]: The missing_dso_whitelist operates as a string match.  If $RPATH/blah is what conda-build is complaining about, you can try 
```suggestion
    - $RPATH/ld-linux.so.2  # [linux]
```

Comment (oblute) [reply recipes/pyclustering/meta.yaml]: @msarahan thanks, trying that!

File: .ci_support/osx64.yaml [hunk 2]
```diff
@@ -6,3 +7,28 @@ fortran_compiler:
   - gfortran
 channel_sources:
   - conda-forge,defaults
+=======
+CONDA_BUILD_SYSROOT:                                                                                    # [osx]
+  - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  # [osx]
+
+<<<<<<< HEAD
```
Comment (msarahan) [inline .ci_support/osx64.yaml]: merge conflict - need to clean up here

File: recipes/pyclustering/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+requirements:
+  build:
+    # - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    # - patchelf
+    # - glib
+  host:
+    - pip
+    - python >=3.5
```
Comment (msarahan) [inline recipes/pyclustering/meta.yaml]: conda-build is a bit non-ideal and does not like version constraints on python here.  Please make this plain "python" and instead add:

```
  skip: True  # [py<35]
```

to the "build" section above.

**Comment by oblute** [comment]
@msarahan moved this to a new PR due to branch issues - linked here https://github.com/conda-forge/staged-recipes/pull/10407 - still having OSX issues only.

**Pull request closed without merging**